docs/configuration-setup.rst

```diff
@@ -0,0 +1,193 @@
+Configure Dask-Jobqueue
+=======================
```

Comment:
Its probably worth adding a link on this page to main dask configuration section. Just mentioning that these concepts are also described there.